bin/butane.js

```diff
@@ -1,9 +1,3 @@
 #!/usr/bin/env node
 
-var argv = require('minimist')(process.argv.slice(2));
-var fireRules = require('../dist/');
-
-var input = argv._[0];
-var output = argv._[1];
-
-fireRules.convert(input, output);
+require('require-relative-main')('./cli', __dirname)
```

Comment:
@bendrucker what's the best way to chat with you — think there might be a bug in `require-relative-main`...